AraEvent/AraEventCalibrator.cxx

```diff
@@ -1213,6 +1241,9 @@ Bool_t AraEventCalibrator::TimingCalibrationAndBadSampleReomval(UsefulAtriStatio
                 std::vector<int> tempSamps;
                 tempSamps = sampleList->at(chanId);
                 sampleList->at(chanId).clear();
+                std::vector<int> tempBlocks;
+                tempBlocks = blockList->at(chanId);
+                blockList->at(chanId).clear();
            
                 int numBlock = int(numPoints/samples_per_block); 
```

Comment:
shouldn't this be the ceiling? what happens to the leftover samples?

Reply:
Sorry I'm not sure I understand this comment? This is just mimicking what is done for `sampleList`